src/components/WebexSettings/README.md

```diff
@@ -0,0 +1,37 @@
+# Webex Settings Component
+
+Webex Settings component displays a tab menu that allows for control over device selection by performing a specific action.
+
+<p align="center">
+  <img src="./WebexSettings.png" alt="Default Webex Seetings" />
```

Comment:
- this image does not seem to exist. Maybe just add the example image from the story for now.
- Seetings -> Settings
- simpler explanation, for example: Webex Settings component allows the user to choose which camera and microphone to use during the meeting